subjects/make-your-game/score/README.md

```diff
@@ -0,0 +1,37 @@
+## score-board
+
+### Objectives
+
+You must follow the same [principles](https://public.01-edu.org/subjects/make-your-game/README.md) as the first subject.
+
+For this project you must take into account the usage of score boards.
+
+The score board must present the following tasks:
+
+- **Countdown clock**: That will indicate the amount of time the player has until the game ends
+- **Score**: That will keep a score count of the amount of XP/points that the player got while playing
+- **Lives**: That indicates the amount of lives that the player has left
```

Comment:
I think this is to be expected in the normal subject, the scoreboard is about having a server to serve highscores, not the UI to display current score.